### PR TITLE
fix(task-framework): remove useless saving results in case of updating status failure

### LINKED ISSUE #3556: [Bug]: Global Terminate button, unsuccessful
### ODC version

4.3.2

### OB version

no related

### What happened?

Global Terminate button, unsuccessful

### What did you expect to happen?

Global stop button, normally available

### How can we reproduce it (as minimally and precisely as possible)?

Select a running task and click the Global Terminate button to observe the system response

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by MarkPotato777**
#### What type of PR is this?

type-bug
#### What this PR does / why we need it:
1. Remove useless saving results in case of updating status failure. `schedule_task.result_json` is a deprecated column, which may cause updating failure because `text` is short for the result value.
2. In the Proccess mode, skip saving log metadata.
#### Which issue(s) this PR fixes:

Fixes #3556 

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by yhilmare — APPROVED** [review]
lgtm

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/listener/DefaultJobProcessUpdateListener.java [hunk 1]
```diff
@@ -58,44 +51,12 @@ public void onEvent(DefaultJobProcessUpdateEvent event) {
...
-        if (taskResult.getExecutorEndpoint() != null) {
-            UriComponents uc = UrlUtils.getUriComponents(taskResult.getExecutorEndpoint());
-            ExecutorInfo executorInfo = new ExecutorInfo();
-            executorInfo.setHost(uc.getHost());
-            executorInfo.setPort(uc.getPort());
-            taskEntity.setExecutor(JsonUtils.toJson(executorInfo));
-        }
-        scheduleTaskService.update(taskEntity);
+    private void updateScheduleTask(ScheduleTask taskEntity) {
+        scheduleTaskService.updateStatusById(taskEntity.getId(), taskEntity.getStatus());
```
Comment (LioRoger) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/listener/DefaultJobProcessUpdateListener.java:59]: change this method name to updateScheduleTaskStatus and the following debug log print new status

Comment (MarkPotato777) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/listener/DefaultJobProcessUpdateListener.java]: done

**Review by LioRoger — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**